shuffle/cli/src/shared.rs

```diff
@@ -478,12 +478,16 @@ pub fn normalized_network_name(network: Option<String>) -> String {
     }
 }
 
-#[derive(Debug, Serialize, Deserialize, PartialEq)]
+#[derive(Debug, Serialize, Deserialize, PartialEq, Clone)]
 pub struct NetworksConfig {
     networks: BTreeMap<String, Network>,
 }
 
 impl NetworksConfig {
+    pub fn _new(networks: BTreeMap<String, Network>) -> NetworksConfig {
```

Comment:
Why the `_new` underscore prefix? Remove please

Reply:
This actually fails the cargo clippy test since it's not used anywhere in the src folder. That `new()` is useful in the cli tests tho, so I put a _ before `new` to make clippy happy.

Reply:
Rather than place a _prefix, let's add a clippy macro to stop it from complaining.

```rust
    #[allow(dead_code)]
    pub fn new(networks: BTreeMap<String, Network>) -> NetworksConfig {
```